TFUtil.py

```diff
@@ -3869,6 +3869,8 @@ def error():
         f = ns[s + "_initializer"]()
       elif s in ns:
         f = ns[s]()
+    elif isinstance(s, numpy.ndarray):
+      f = ns["Constant"].from_config({'value': s, 'verify_shape': True})
```

Comment:
Why so complicated and indirect?
Just in the beginning of this function, after `if isinstance(s, (float, int))`, just add this case, and directly use `tf.constant_initializer`.
Then you also don't need other changes in this function.

Reply:
Right. From the online documentation I got the impression that the option `verify_shape` wouldn't be available this way but sure it is and that's nicer.